src/imgui/ImGuiDebugger.cc

```diff
@@ -496,60 +501,14 @@ void ImGuiDebugger::drawDisassembly(CPURegs& regs, MSXCPUInterface& cpuInterface
 							if (!bpRightClick && ImGui::IsItemClicked(ImGuiMouseButton_Right)) {
 								ImGui::OpenPopup("disassembly-context");
 							}
-							auto addrStr = tmpStrCat(hex_string<4>(addr));
+							if (manager.getShortcuts().checkShortcut(ShortcutIndex::GOTO_DISASM)) {
+								ImGui::OpenPopup("disassembly-context");
+							}
```

Comment:
This is fine for this PR. But it's not OK for the final code (needs to be fixed later).

The idea of using keyboard shortcuts is to not have to switch between keyboard and mouse.
So after pressing Ctrl+G (activate GOTO_DISASM) it should be possible to immediately start typing the new address.

But instead now it only opens the context menu. And you still have to click in the correct field before you can start typing.
In the future GOTO_DISASM should also focus the correct field.